tests/api/route-services/samples.test.js

```diff
@@ -149,4 +148,58 @@ describe('tests for the samples service', () => {
       })
       .then(() => done());
   });
+
+  it('addSample works', (done) => {
```

Comment:
Is there any case where `addSample` doesn't work? Do we need to test for it?

Reply:
See https://github.com/biomage-ltd/api/pull/231#discussion_r722684737